SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialOSWorkerInterface/SpatialOSWorkerConnectionSpy.cpp

```diff
@@ -82,6 +109,12 @@ Worker_RequestId SpatialOSWorkerConnectionSpy::SendEntityQueryRequest(const Work
 	return NextRequestId++;
 }
 
+Worker_RequestId SpatialOSWorkerConnectionSpy::SendEntityQueryRequest(const Worker_EntityQuery* EntityQuery,
+																	  SpatialGDK::FRetryData RetryDAta)
```

Comment:
Typo - s/DAta/Data

Reply:
Also if RetryData isn't gonna be used, variable name should be commented out to prevent warnings.